charts/spire/charts/spire-server/values.yaml

```
@@ -86,6 +86,12 @@ trustDomain: example.org
 
 bundleConfigMap: spire-server
 
+federation:
+  bundleEndpoint:
+    enabled: true
```

```suggestion
    enabled: false
```

I think false is a better default.

Please also add a test scenario in `.github/tests/federation` where federation is enabled.

As well a test in `charts/spire/charts/spire-server/templates/tests` to test the federation endpoint.